serverless/deploy/github/distributor/update_logs_index/main.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// update_logs_index is a tool to manage a markdown file containing a list of
+// human-readable links to the log directories maintained by the distributor.
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/google/trillian-examples/serverless/config"
+	"golang.org/x/mod/sumdb/note"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"path/filepath"
+	"text/template"
+
+	i_note "github.com/google/trillian-examples/internal/note"
+)
+
+var (
+	storageDir = flag.String("distributor_dir", "", "Root directory of the distributor.")
+	configFile = flag.String("config", "", "Distributor config file.")
+	outputFile = flag.String("output", "log_index.md", "Markdown file to write.")
+)
+
+const tmpl = `# {{.Title}}
+
+## Known logs
```

Comment:
nit: I'd prefer Logs to be title-case (similarly, Witnesses below)

Reply:
sure, done